client/app/api/Base.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+export default class BaseAPI {
+  constructor() {
+    this.client = null;
+  }
+
+  /** Returns the API client */
+  getClient() {
```

Comment:
Here got some duplications with `lib/axios`, but I am going to remove `lib/axios` after all the code starting using the api layer to talk to the server.